src/github.com/getlantern/tarfs/tarfs.go

```diff
@@ -252,7 +252,7 @@ func (f *AssetFile) Readdir(count int) ([]os.FileInfo, error) {
 }
 
 func (f *AssetFile) Stat() (os.FileInfo, error) {
-	return f, nil
+	return &f.FakeFile, nil
```

Comment:
I don't think so, though actually I don't know what causes the ambiguity. Removing `AssetDirectory` altogether doesn't stop this error.